ROADMAP.md

```diff
@@ -0,0 +1,19 @@
+# CNI Roadmap
+
+This document defines a high level roadmap for CNI development.
+The list below is not complete, and we advise to get the current project state from the [milestones defined in GitHub](https://github.com/containernetworking/cni/milestones).
+
+## CNI Milestones
+
+### [v0.3.0](https://github.com/containernetworking/cni/milestones/v0.3.0)
```

Comment:
peanut gallery, this seems a bit uselessly vague. how about an estimated timeline and/or at least some high level goals for the milestone releases?